packages/react-charting/src/components/SankeyChart/SankeyChart.base.tsx

```diff
@@ -526,7 +528,7 @@ function computeNodeAttributes(
       reactId: getId('nodeBar'),
       gElementId: getId('nodeGElement'),
       name: truncatedname,
-      aria: nodeAriaLabel(singleNode, nodeValue),
+      aria: nodeAriaLabel(singleNode, nodeValue, formatNumber),
```

Comment:
this parameter should be optional in all places

Reply:
updated to remove parameter and call the private method `_formatNumber` directly in `nodeAriaLabel` method definition